ivy/functional/frontends/torch/nn/functional/convolution_functions.py

```diff
@@ -57,7 +55,18 @@ def _conv_transpose(
     padding, output_padding = map(
         lambda x: [x] * dims if isinstance(x, int) else x, [padding, output_padding]
     )
-    pad_widths = [(weight.shape[i] - 1,) * 2 for i in range(dims)]
+    dilation = [dilation] * dims if isinstance(dilation, int) else dilation
+    stride = [stride] * dims if isinstance(stride, int) else stride
```

Comment:
The `map` above is doing exactly that. You could just add `dilation` and `stride` there to avoid repetition.